sql/src/main/java/io/crate/planner/operators/ExplainLogicalPlan.java

```diff
@@ -170,7 +170,17 @@ public Context(PlannerContext plannerContext, ProjectionBuilder projectionBuilde
         }
 
         @Override
-        public ImmutableMap.Builder<String, Object> visitJoin(Join logicalPlan, Context context) {
+        public ImmutableMap.Builder<String, Object> visitNestedLoopJoin(NestedLoopJoin logicalPlan, Context context) {
+            return createMap(logicalPlan, createSubMap()
+                .put("left", explainMap(logicalPlan.lhs, context))
+                .put("right", explainMap(logicalPlan.rhs, context))
+                .put("joinType", logicalPlan.joinType())
+                .put("joinCondition",
+                    SymbolPrinter.INSTANCE.print(logicalPlan.joinCondition(), SymbolPrinter.Style.FULL_QUALIFIED)));
```

Comment:
could be extracted and used by both join plans

Reply:
hm no.... No superclass to call the lhs, rhs, joinType, etc...